tracks/python/exercises/pig-latin/pig-latin/mentoring.md

```diff
@@ -0,0 +1,87 @@
+# Mentoring
+
+## Problem and challenges
+
+The problem requires the student to rearrange the English word based on the given rules in the instruction.
+
+## Reasonable solutions
+
+This exercise can be solved by using one `for` loop.
+
+```
+def translate(text):
+    return ' '.join([pig(word) for word in text.split(' ')])
+def pig(word):
```

Comment:
Functions should be separated by two blank lines.

Reply:
okay.